addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -225,8 +235,17 @@ func _submit_input(input: Dictionary, tick: int):
 		sanitized[property] = value
 	
 	if sanitized.size() > 0:
-		_inputs[tick] = sanitized
-		_earliest_input = min(_earliest_input, tick)
+		for property in sanitized:
+			for i in range(0, sanitized[property].size()):
+				var t = tick - i
+				var old_input = _inputs.get(t, {}).get(property)
+				var new_input = sanitized[property][i]
+				
+				if old_input == null:
```

Comment:
There's a tiny window for cheating if we allow re-writing of what's previously been received with

```
if old_input != new_input:
```

so changed it to only allow backfilling missing inputs.

Reply:
Nice catch!